terraform/pvsite/development/main.tf

```diff
@@ -61,7 +61,7 @@ module "pvsite_ecs" {
 }
 
 module "pvsite_forecast" {
-  source = "../../modules/services/forecast_pvsite"
+  source = "../../modules/services/forecast_generic"
```

Comment:
do you need to add datadir in here, or is there a default one

Reply:
ah maybe you do it in the variables

Reply:
That gets the nwp_bucket_config as a variable from TF Cloud itself - so yes I do, but to do that I updated the variable in TF Cloud